x-pack/plugins/translations/translations/ja-JP.json

```diff
@@ -138,16 +138,6 @@
     "charts.controls.rangeErrorMessage": "値は{min}と{max}の間でなければなりません",
     "charts.controls.vislibBasicOptions.legendPositionLabel": "凡例位置",
     "charts.controls.vislibBasicOptions.showTooltipLabel": "ツールヒントを表示",
-    "common.ui.errorUrlOverflow.breadcrumbs.errorText": "エラー",
```

Comment:
I changed the defaultMessage of these strings significantly, so I figured I should delete the old translations so they get updated. Is that the right process @Bamieh ?